rules/windows/process_creation/proc_creation_win_uac_bypass_trustedpath.yml

```diff
@@ -16,8 +21,13 @@ logsource:
     product: windows
 detection:
     selection:
-        Image|contains: 'C:\Windows \System32\'
+        - Image|contains:
+              - ':\Windows \System32\'
+              - ':\Windows \SysWOW64\'
+        - CommandLine|contains:
+              - ':\Windows \System32\' # Could be file copy operation to/from this path
+              - ':\Windows \SysWOW64\'
```

Comment:
This could introduce FPs and is not needed in the context of the rule, because the image already covers the technique.